packages/manager/modules/pci/src/projects/project/instances/instances.html

```diff
@@ -130,7 +130,7 @@ <h1 data-translate="pci_projects_project_instances_title"></h1>
             <oui-action-menu-item
                 data-text="{{:: 'pci_projects_project_instances_hardReboot_label' | translate }}"
                 data-on-click="$ctrl.hardRebootInstance($row)"
-                data-ng-if="$row.status === 'ACTIVE'">
+                data-ng-if="$row.status === 'ACTIVE' || $row.status === 'SHUTOFF'">
```

Comment:
```suggestion
                data-ng-if="['ACTIVE', 'SHUTOFF'].includes($row.status)">
```

Reply:
```suggestion
                data-ng-if="$row.canBeHardRebooted()>
```

Reply:
I think it should be updated too in instance detail : https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/instance/instance.html#L46

Reply:
done thanks